### PR TITLE
Clean Up Root Directory

### PR DESCRIPTION
**Pull request by rosborne132**
**CHANGELOG** :memo:

* Updated README.md to point to new folders
* Update .gitignore to ignore unneeded files.

**PRINTS(OPTIONAL)** :framed_picture:

**LABELS**

* enhancement
* hacktoberfest

### CODE REVIEW

**Comment by pyymenta** [comment]
@rosborne132 Thank you for your contribuition! I'll review it!

File: .gitignore [hunk 1]
```diff
@@ -1 +1,14 @@
...
 node_modules
+
+# OS metadata
+.DS_Store
+
+# Builds
+dist
+
+# Lock files
+yarn.lock
```
Comment (pyymenta) [inline .gitignore]: I think we shouldn't ignore the `yarn.lock` file for the same reason we don't ignore the `package-lock.json` file. Could you remove this line?

Comment (rosborne132) [reply .gitignore]: @pyymenta  Work has been updated. Thanks for taking a look at this!

**Review by pyymenta — APPROVED** [review]
LGTM 🚀 🚀

**Comment by rosborne132** [comment]
@pyymenta Can we update the labels to include `hacktoberfest-accepted`? :)

**Pull request merged**